.github/workflows/pr-lint.yml

```diff
@@ -15,3 +15,22 @@ jobs:
       - uses: amannn/action-semantic-pull-request@v5
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          scopes: |
```

Comment:
IMO we should add `deps` to this list

Reply:
You're right! I've added it to the spec as it was missing.